Microservices/ServiceRegistryMicroservice/src/main/resources/application.properties

```diff
@@ -0,0 +1,26 @@
+spring.application.name=ServiceRegistryMicroservice
+# Create variable
+DB_PORT=3306
+DB_NAME=big_data
+DB_IP=localhost
+SAMPLE_MICROSERVICE_PORT=8090
+MYSQL_USERNAME=root
+MYSQL_PASSWORD=
+# Database
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.url = jdbc:mysql://${DB_IP}:${DB_PORT}/${DB_NAME}
+spring.datasource.username = ${MYSQL_USERNAME}
+spring.datasource.password = ${MYSQL_PASSWORD}
+spring.jpa.show-sql = true
+spring.jpa.hibernate.ddl-auto = update
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQLDialect
+server.port=${SAMPLE_MICROSERVICE_PORT}
+# Liquibase
+spring.liquibase.enabled=true
+# NOTE: If you want to use liquibase, you can set it to true, otherwise it will not validate the schema
+spring.liquibase.change-log=classpath:/db/changelog-master.xml
+spring.liquibase.database-change-log-lock-table=LOCK_TABLE
+spring.liquibase.database-change-log-table=DATABASE_CHANGE_LOG
+logging.level.liquibase=DEBUG
+logging.file.name=logs/app.log
+# File where we can store the logs
```

Comment:
Can you remove all the comments here, the secrets can stay until we have .env setup